modules/update-availability-faq.adoc

```diff
@@ -35,11 +35,10 @@ The primary purpose of the `eus` channel is to serve as a convenience for cluste
 * A release that is available on the `fast` channel always becomes available on the `stable` channel after this delay.
 
 [id="supported-updates_{context}"]
-*What does it mean if an update is supported but not recommended?*
+*What does it mean if an update has known issues?*
 
-* Red Hat continuously evaluates data from multiple sources to determine whether updates from one version to another lead to issues.
-If an issue is identified, an update path may no longer be recommended to users.
-However, even if the update path is not recommended, customers are still supported if they perform the update.
+* Red Hat continuously evaluates data from multiple sources to determine whether updates from one version to another have any declared issues. Identified issues are typically documented in the version's release notes.
```

Comment:
This is out of scope to your changes but per the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces) there should be a non breaking space in between Red and Hat:
```suggestion
* Red{nbsp}Hat continuously evaluates data from multiple sources to determine whether updates from one version to another have any declared issues. Identified issues are typically documented in the version's release notes.
```

Reply:
nice catch!